src/esp/physics/PhysicsManager.h

```diff
@@ -1025,41 +1025,59 @@ class PhysicsManager : public std::enable_shared_from_this<PhysicsManager> {
     return false;
   };
 
+  /**
+   * @brief Perform discrete collision detection for the scene with the derived
+   * PhysicsManager implementation. Not implemented for default @ref
+   * PhysicsManager. See @ref bullet::BulletPhysicsManager.
+   */
+  virtual void performDiscreteCollisionDetection() {
```

Comment:
Should this be a pure virtual? = 0?

Reply:
We instance the base PhysicsManager in the absence of Bullet, so it cannot be and abstract class.